frontend/src/components/SpectatorSideBar/index.jsx

```diff
@@ -51,6 +51,40 @@ const SpectatorSideBar = ({ type, gameData }) => {
   const [commentMsg, setCommentMsg] = useState('');
   const [commentsFromGameData] = useState(gameData.messages);
 
+  const mapTimeStamp = (timeStamp) => {
```

Comment:
Please move all this logic into a utility function

Reply:
Done, Sir!